include/zephyr/devicetree.h

```diff
@@ -27,6 +27,7 @@
 /**
  * @brief devicetree.h API
  * @defgroup devicetree Devicetree
+ * @since 2.0.0
```

Comment:
The entire DT API was rewritten completely for v2.3.0; this is definitely one of the cases where the automation is not working (I see that's explained in the commit message), but it does mean this content is incorrect.

Reply:
If the DT API was there on version 2.0.0, but we need to track it as v2.3.0 because of a rewrite, then all bets are off for automation =D
The idea of the automated search was to provide some base - which would have errors, that hopefully reviewers with more historical knowledge would help fix - if not on this PR, over some time after it gets in. It seems fairly good for more recent stuff, but yeah, older APIs have more history to them.
My feeling then is that the best way here is to drop the results from the automated search and only add the ones that were present on the original overview.rst - thoughts?

Reply:
> My feeling then is that the best way here is to drop the results from the automated search

Yeah, removing the automation makes sense to me